currency/work.cc.lua

```diff
@@ -0,0 +1,69 @@
+{{/*
+		Simple Currency System By GasInfinity
+        
+        Recommended Trigger: Command (mention/cmd prefix): "work"
+        
+        Simple Work Command With Configurable Msg and configurable minWinAmount and maxWinAmount, Usage "!work" 
+*/}}
+
+
+{{/*CONFIGURATION START*/}}
+{{ $dbHandName := "HAND" }} {{/*Database Currency Entry*/}}
+{{ $currency := "💰" }} {{/*Currency Emoji/Name*/}}
+{{ $minAmount := 5}} {{/*Minimum Money Given*/}}
+{{ $maxAmount := 50}} {{/*Maximum Money Given*/}}
+{{$cooldown := 3600}} {{/*Cooldown "In Seconds": 60secs = 1 min, 3600secs = 1h, 86400 = 1day*/}}
+{{ $winMsg := "You Worked Hard And Gained: "}}
+{{ $loseMsg := "You Losed Your Hammer and Payed For Replace it: "}}
+{{/*CONFIGURATION END*/}}
+
+
+{{/*MAIN CODE!!!*/}}
+
+{{/*COOLDOWN CODE*/}}
+{{/* 0 for per user, 1 for global */}}
+{{$isGlobal := 0}}
+{{$name := "work"}}
+{{$lengthSec := $cooldown}}
+{{$error4 := "**Hey, This Command have a Cooldown of: "}}

```

Comment:
You can write cooldown logic for this CC if you're up to it, would be nice.

Reply:
doing